java_tests/src/main/java/org/infinispan/client/hotrod/configuration/ServerConfigurationBuilder.java

```diff
@@ -8,11 +8,10 @@
  * @author Tristan Tarrant
  * @since 5.3
  */
-public class ServerConfigurationBuilder extends AbstractConfigurationChildBuilder implements Builder<ServerConfiguration> {
+public class ServerConfigurationBuilder {
```

Comment:
Martin you're right, these changes were for rel 7.0. Let me see if I can keep the cleanup preserving the api